<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modDepart</class>
 <widget class="QDialog" name="modDepart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Depart/Modifier</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Res.qrc">
    <normaloff>:/icones/Icones/depart.jpg</normaloff>:/icones/Icones/depart.jpg</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Motif depart</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPlainTextEdit" name="motifdepartEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Date depart</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDateEdit" name="datedepartEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="typedepartCombo"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Type depart</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Agent</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="agentEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btAgent">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../Res.qrc">
           <normaloff>:/icones/Icones/add.ico</normaloff>:/icones/Icones/add.ico</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btValider">
       <property name="text">
        <string>Valider</string>
       </property>
       <property name="icon">
        <iconset resource="../Res.qrc">
         <normaloff>:/icones/Icones/15-Tick-icon.png</normaloff>:/icones/Icones/15-Tick-icon.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btFermer">
       <property name="text">
        <string>Fermer</string>
       </property>
       <property name="icon">
        <iconset resource="../Res.qrc">
         <normaloff>:/icones/Icones/gnome_application_exit.png</normaloff>:/icones/Icones/gnome_application_exit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btFermer</sender>
   <signal>clicked()</signal>
   <receiver>modDepart</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>493</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
